src/einsteinpy/symbolic/tensor.py

```diff
@@ -121,6 +122,24 @@ def config(self):
     def __getitem__(self, index):
         return self.arr[index]
 
+    def __str__(self):
+        """
+        Returns a String with a readable representation of the object of class Tensor
+
+        """
+        representation = "Tensor" + "\n"
+        representation += self.arr.__str__()
```

Comment:
It should be noted that this does not support the pretty printing that sympy is designed for. Meaning that interactive environments will have a hard time understanding the values returned by these objects.

Reply:
I know it would render as plaintext in the IPython notebook. Would keep it in mind. Meanwhile pretty printers can do `obj.tensor()`.